lib/Test/Pod/CoverageChange.pm

```diff
@@ -0,0 +1,246 @@
+use strict;
+use warnings;
+
+package Test::Pod::CoverageChange;
+# ABSTRACT: check perl modules against their pod coverage
+
+our $VERSION = '0.001';
+# AUTHORITY
+
+use utf8;
+
+=encoding utf8
+
+=head1 NAME
+
+Pod coverage calculator test file.
+
+=head1 SYNOPSIS
+
+    use Test::Pod::CoverageChange qw(pod_coverage_syntax_ok);
+
+    Test::Pod::CoverageChange::pod_coverage_syntax_ok('lib', {
+        Module::With::3::expected::naked::subs              => 3,
+        AnotherModule::With::10::expected::naked::subs      => 10,
+        YetAnotherModule::With::1::expected::naked::subs    => 1,
+        YetAnotherModule::With::5::expected::naked::subs    => 5,
+    }, [
+        We::Ignore::ThisModule,
+        We::Also::Ignore::This::Module
+    ]);
+
+=head1 DESCRIPTION
+
+It checks all files that placed under a given directory against their
+POD coverage to see if all existing subs have POD and also
+POD syntax to see if is there an POD syntax error or not?
+
+Prints the percentage of POD coverage in B<TODO> test format for the packages that we allowed to have naked subs.
```

Comment:
This should never *print* anything, it *reports* and the TAP formatter handles output